packages/common/src/Registry.ts

```diff
@@ -4,8 +4,9 @@ import { mkdir, writeFile } from 'node:fs/promises';
 import path from 'pathe';
 import { tsImport } from 'tsx/esm/api';
 import { ConfigStore, ConfigStoreConstructor, Expression, ExpressionFunction } from '@configu/sdk';
+import { getConfiguHomeDir } from './utils';
 
-const CONFIGU_HOME = path.join(process.cwd(), '/.configu-cache');
+const CONFIGU_HOME = path.join(getConfiguHomeDir(), '/.cache');
```

Comment:
Ran's comment: rename cache to `integrations` or `stores`

Reply:
Decided to rename `cache` to `integrations` for future-proofing in case we ever decide to remote register something other than stores (expressions, etc..)